### PR TITLE
chore: schedule release

### PR DESCRIPTION
**Pull request by eunjae-lee**
## Summary

This PR schedules release every Tuesday 11am (Paris time). It doesn't automatically releases the new version, but just calls `shipjs prepare` to create a release pull-request.

### CODE REVIEW

File: ship.config.js [hunk 1]
```diff
@@ -2,6 +2,13 @@ const fs = require("fs");
 const path = require("path");
 
 module.exports = {
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
```
Comment (eunjae-lee) [inline ship.config.js:7]: If it's a patch update and there is no fix (which means only `chore` commits), then we do not prepare a release pull-request. This prevents unnecessary pull-requests every week.

File: .circleci/config.yml [hunk 2]
```diff
@@ -99,3 +113,13 @@ workflows:
...
+  prepare_release_every_tuesday:
+    triggers:
+      - schedule:
+          cron: '0 9 * * 2'
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - prepare release
```
Comment (JonathanMontane) [inline .circleci/config.yml:125]:
```suggestion
      - prepare release

```

File: .circleci/config.yml [hunk 3]
```diff
@@ -86,6 +86,20 @@ jobs:
...
+    steps:
+      - checkout
+      - run: *install_yarn_version
+      - restore_cache: *restore_yarn_cache
+      - run: *run_yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Prepare release
+          command: |
+            git config --global user.email "eunjae.lee@algolia.com"
```
Comment (JonathanMontane) [inline .circleci/config.yml:100]: We could put any name here right? If that's the case, I would advocate for no PII if possible, so something like `events-ingestion-team@algolia.com`.

File: ship.config.js (hunk 1 above)
Comment (JonathanMontane) [reply ship.config.js:7]: So if I understand correctly, this will prevent Renovate from triggering new release PRs?

**Review by JonathanMontane — APPROVED** [review]
LGTM 🚢 
Might be best to have no PIIs but not a blocker for me (as it's your PII).

File: ship.config.js (hunk 1 above)
Comment (eunjae-lee) [reply ship.config.js:7]: Even before `shouldPrepare` is called, Ship.js checks if the latest commit message starts with "chore: release v", and thus renovate commits won't trigger new releases.

File: .circleci/config.yml (hunk 3 above)
Comment (eunjae-lee) [reply .circleci/config.yml:100]: Yeah actually, the InstantSearch team has a shared email address for this purpose. 

something like `events-ingestion@algolia.com / Algolia`? Not sure if that email address is suitable for this process 🤔

Comment (eunjae-lee) [reply .circleci/config.yml:100]: @JonathanMontane 

https://github.com/algolia/places/blob/38e8f30fd55cecdc723f096abd8c225e335f4e81/.travis.yml#L4-L5

what about that?

Comment (JonathanMontane) [reply .circleci/config.yml:100]: probs okay 🙂

**Comment by eldeloro75017** [comment]
> ## Summary
> This PR schedules release every Tuesday 11am (Paris time). It doesn't automatically releases the new version, but just calls `shipjs prepare` to create a release pull-request.

**Comment by eldeloro75017** [comment]
awwwyeah

**Pull request merged**